cht-petals/requirements.txt

```
@@ -1,9 +1,9 @@
-fastapi==0.95.0
-uvicorn==0.21.1
-pytest==7.2.2
-requests==2.28.2
-tqdm==4.65.0
-httpx==0.23.3
-python-dotenv==1.0.0
-tenacity==8.2.2
-petals==2.2.0
+fastapi==0.*
+uvicorn==0.*
+pytest==7.*
+requests==2.*
+tqdm==4.*
+httpx==0.*
```

```suggestion
fastapi==0.95.*
uvicorn==0.21.*
pytest==7.*
requests==2.*
tqdm==4.*
httpx==0.23.*
```

Because https://semver.org